boards.txt

```diff
@@ -23243,3 +23243,166 @@ crabik_slot_esp32_s3.menu.EraseFlash.all=Enabled
 crabik_slot_esp32_s3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+
+# ESP32-S3 PowerFeather
+
+esp32s3_powerfeather.name=ESP32-S3 PowerFeather
+esp32s3_powerfeather.vid.0=0x303a
+esp32s3_powerfeather.pid.0=0x81BB
+
+esp32s3_powerfeather.bootloader.tool=esptool_py
+esp32s3_powerfeather.bootloader.tool.default=esptool_py
+
+esp32s3_powerfeather.upload.tool=esptool_py
+esp32s3_powerfeather.upload.tool.default=esptool_py
+esp32s3_powerfeather.upload.tool.network=esp_ota
+
+esp32s3_powerfeather.upload.maximum_size=1310720
+esp32s3_powerfeather.upload.maximum_data_size=327680
+esp32s3_powerfeather.upload.flags=
+esp32s3_powerfeather.upload.extra_flags=
+esp32s3_powerfeather.upload.use_1200bps_touch=true
+esp32s3_powerfeather.upload.wait_for_upload_port=true
+
+esp32s3_powerfeather.serial.disableDTR=false
+esp32s3_powerfeather.serial.disableRTS=false
+
+esp32s3_powerfeather.build.tarch=xtensa
+esp32s3_powerfeather.build.bootloader_addr=0x0
+esp32s3_powerfeather.build.target=esp32s3
+esp32s3_powerfeather.build.mcu=esp32s3
+esp32s3_powerfeather.build.core=esp32
+esp32s3_powerfeather.build.variant=esp32s3_powerfeather
+esp32s3_powerfeather.build.board=ESP32S3_POWERFEATHER
+
+esp32s3_powerfeather.build.usb_mode=0
```

Comment:
You probably want usb_mode to be 1 by default and use the hardware CDC+JTAG. It's the preferred way if you do not use a special bootloader that would require TinyUSB